osm/anaconda-project.yml

```diff
@@ -50,10 +50,11 @@ env_specs:
     packages:
       - holoviews
       - intake
-      - intake-parquet
       - ipywidgets
       - jupyter
       - tqdm
+      - pip:
+        - git+https://github.com/intake/intake-parquet.git@jsignell/spatial#egg=intake_parquet
```

Comment:
intake-parquet isn't on conda?

Reply:
This points to a branch. The spatial isn't merged or released yet.